externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/PlanOnlyStatement.scala

```diff
@@ -38,23 +38,25 @@ class PlanOnlyStatement(
   private val operationLog: OperationLog = OperationLog.createOperationLog(session, getHandle)
   override def getOperationLog: Option[OperationLog] = Option(operationLog)
 
-  override protected def resultSchema: StructType =
-    if (result == null) {
+  override protected def resultSchema: StructType = Option(result) match {
```

Comment:
can we avoid creating an Option here?

Reply:
Of course it is possible, here is just my personal scala programming preference, try to avoid dealing with `null`, and not to produce `null`. Is it sure that the wrapping of Option needs to be removed here?

Reply:
we can remove changes here